app/assets/javascripts/modules/track-radio-group.js

```diff
@@ -1,95 +1,104 @@
+window.GOVUK.TrackRadioGroupVerify = function (data) {
+  window.GOVUK.triggerEvent(document, 'have-verify-data', { detail: data })
+};
 
-  var $ = global.jQuery
+(function (Modules) {
+  function TrackRadioGroup (element) {
+    this.element = element
+  }
 
-  GOVUK.Modules.TrackRadioGroup = function () {
-    this.start = function (element) {
-      track(element)
+  TrackRadioGroup.prototype.init = function () {
+    document.addEventListener('have-verify-data', function (event) {
+      this.reportVerifyUser(this.element, event.detail)
+    }.bind(this))
 
-      if (crossDomainTrackingEnabled(element)) {
-        addCrossDomainTracking(element)
-      }
+    this.track(this.element)
 
-      checkVerifyUser(element)
+    if (this.crossDomainTrackingEnabled(this.element)) {
+      this.addCrossDomainTracking(this.element)
     }
 
-    function track (element, withHint) {
-      element.on('submit', function (event) {
-        var options = { transport: 'beacon' }
+    this.checkVerifyUser(this.element)
+  }
 
-        var $submittedForm = $(event.target)
+  TrackRadioGroup.prototype.track = function (element, withHint) {
+    this.element.addEventListener('submit', function (event) {
+      var options = { transport: 'beacon' }
+      var submittedForm = event.target
+      var checkedOption = submittedForm.querySelector('input:checked')
+      var eventValue = this.eventTrackingValue(checkedOption, withHint)
 
-        var $checkedOption = $submittedForm.find('input:checked')
+      GOVUK.analytics.trackEvent('Radio button chosen', eventValue, options)
 
-        var eventValue = eventTrackingValue($checkedOption, withHint)
+      if (this.crossDomainTrackingEnabled(submittedForm)) {
+        this.trackCrossDomainEvent(submittedForm, eventValue, options)
+      }
+    }.bind(this))
+  }
 
-        GOVUK.analytics.trackEvent('Radio button chosen', eventValue, options)
+  TrackRadioGroup.prototype.checkVerifyUser = function (element) {
+    var url = 'https://www.signin.service.gov.uk/hint'
+    var timestamp = Math.floor(Date.now() / 1000)
+    var script = document.createElement('script')
+    script.setAttribute('src', url + '?callback=window.GOVUK.TrackRadioGroupVerify&_=' + timestamp)
+    document.body.appendChild(script)
+  }
 
-        if (crossDomainTrackingEnabled($submittedForm)) {
-          trackCrossDomainEvent($submittedForm, eventValue, options)
-        }
-      })
-    }
+  TrackRadioGroup.prototype.trackVerifyUser = function (element, data) {
+    this.reportVerifyUser(element, data)
+  }
 
-    function checkVerifyUser (element) {
-      $.ajax({
-        url: 'https://www.signin.service.gov.uk/hint',
-        cache: false,
-        dataType: 'jsonp',
-        timeout: 3000
-      }).then(function (data) {
-        reportVerifyUser(element, data)
-      }, function (e) { console.log('error', e) })
+  TrackRadioGroup.prototype.reportVerifyUser = function (element, data) {
+    if (data != null && data.value === true) {
```

Comment:
This is outside the scope of this PR, but I worry slightly about this line. Am I right in thinking that if Verify suddenly stopped returning anything (i.e. it returned `null`) then none of the analytics events in this module would fire anymore?

Reply:
Note for history. No, if it returned `null` then events would still fire, but the behaviour wasn't quite working as intended. 

Expected behaviour: of the two GA events in the code, only the normal one was firing on live (the cross domain one wasn't happening). When a user submits the form, one event should fire to record that, and a second should fire if the user has previously signed in to Verify. The code now does that.